resources/teams.py

```diff
@@ -170,3 +171,62 @@ def update_team(
             "Server error",
         )
     return team
+
+
+@router.put(
+    "/{team_id}/members",
+    status_code=status.HTTP_200_OK,
+    response_model=List[TeamMember],
+    responses=error_message_models(
```

Comment:
Mangler 401 som mulighet her? Eller får man bare 403 om auth header mangler?

Reply:
Man får vel 403 uten auth, og 401 med invalidert token. Dette gjelder vel i og for seg alle endepunktene, så kan legge til det.